src/tkwidget.jl

```diff
@@ -266,13 +266,15 @@ function reveal(c::Canvas)
 end
 
 @static if is_apple()
-if @compat Sys.WORD_SIZE == 32
-    const CGFloat = Float32
-else
-    const CGFloat = Float64
-end
-objc_msgSend{T}(id, uid, ::Type{T}=Ptr{Void}) = ccall(:objc_msgSend, T, (Ptr{Void}, Ptr{Void}),
-    id, ccall(:sel_getUid, Ptr{Void}, (Ptr{UInt8},), uid))
+    if @compat Sys.WORD_SIZE == 32
+        const CGFloat = Float32
+    else
+        const CGFloat = Float64
```

Comment:
you should still be able to run 32 bit applications though, if it's possible to compile them that way - but I don't think anyone has tried to do so with Julia any time recently